js/ui/codeView.js

```diff
@@ -415,9 +415,9 @@ var CodingSession = GObject.registerClass({
         this._toolboxAppActionGroup.list_actions();
 
         this._overviewHiddenId = Main.overview.connect('hidden',
-                                                       this._syncButtonVisibility.bind(this));
```

Comment:
I would prefer if this commit only added the C class for the effect, and its integration bits from `codeView.js` moved completely into "codeView: Add codeview to provide Flip to Hack".